migrations/900000000000029_sqlq_support_schema.up.sql

```diff
@@ -0,0 +1,42 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS mergestat.jobs_types (
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS mergestat.job_types (
```

suggestion to change the table name